sql/core/src/test/resources/sql-tests/results/group-by-filter.sql.out

```diff
@@ -95,41 +95,49 @@ struct<count(id):bigint>
 
 
 -- !query 9
-SELECT a, COUNT(b) FILTER (WHERE a >= 2) FROM testData GROUP BY a
+SELECT COUNT(DISTINCT id) FILTER (WHERE date_format(hiredate, "yyyy-MM-dd HH:mm:ss") = "2001-01-01 00:00:00") FROM emp
 -- !query 9 schema
-struct<a:int,count(b):bigint>
+struct<count(DISTINCT id):bigint>
 -- !query 9 output
+3
```

Comment:
This result is incorrect too.
It seems the I have to check `date_format`.
@maropu Do you know the reason?

Reply:
I fixed this issue.